Runtime/Scripts/Export/StandardMaterialExport.cs

```diff
@@ -334,6 +352,27 @@ public override bool ConvertMaterial(UnityEngine.Material uMaterial, out Materia
             return success;
         }
 
+        Texture2D ResizeTexture(Texture2D texture, string path, int scale)
+        {
+            int width = texture.width / scale;
+            int height = texture.height / scale;
+
+            RenderTexture renderTexture = new RenderTexture(width, height, 32);
+            RenderTexture.active = renderTexture;
+            Graphics.Blit(texture, renderTexture);
+
+            texture.Reinitialize(width, height);
+            texture.ReadPixels(new Rect(0, 0, width, height), 0, 0);
+            texture.name = Path.GetFileNameWithoutExtension(path);
+
+            bool hasAlpha = GraphicsFormatUtility.HasAlphaChannel(texture.graphicsFormat);
+            byte[] data = hasAlpha ? texture.EncodeToPNG() : texture.EncodeToJPG();
+            string newPath = path.Replace(".", $"_{width}x{height}.");
+            File.WriteAllBytes(newPath, data);
```

Comment:
Can this be removed or do we need to create a new asset?

Reply:
I tested without writing the file out but then the texture wasn't written properly without this. I'll check again and see if I can do everything in memory.

Reply:
OK, I've fixed it :tada: